config/kraft/broker.properties

```diff
@@ -26,8 +26,8 @@ process.roles=broker
 # The node id associated with this instance's roles
 node.id=2
 
-# The connect string for the controller quorum
-controller.quorum.voters=1@localhost:9093
```

Comment:
We still support static quorum, and will for a while. I would suggest commenting the line with `controller.quorum.voters` out, and adding a comment saying that you should comment out either `controller.quorum.voters` or `controller.quorum.bootstrap.servers`, depending on whether you want to use a KIP-853 dynamic quorum or not.

(By default, the uncommented one can be `controller.quorum.bootstrap.servers`, so that more people try the new thing.)

Reply:
This is the broker's configuration in isolated mode. `controller.quorum.voters` is not needed and should not be used in the broker even if they are using it in the controller. On the broker's they should always use `controller.quorum.bootstrap.servers`.

Reply:
ok